src/components/LayerUI.tsx

```diff
@@ -125,7 +125,7 @@ export const LayerUI = React.memo(
           <div className="App-menu App-menu_top">
             <Stack.Col gap={4}>
               <Section heading="canvasActions">
-                <Island padding={4}>
+                <Island padding={4} style={{ zIndex: 1 }}>
```

Comment:
Can you add a comment that explains why the zindex. Going to be less likely to break in the future

Reply:
Right. I'll just link to this PR as the explanation is somewhat involved (check descr).